sys/linux/futex.txt

```diff
@@ -17,4 +18,13 @@ robust_list {
 	next	ptr[in, robust_list, opt]
 }
 
+futex_wait_block {
+	val		int64
```

Comment:
Does syzkaller accepts uint64 instead of int64?

Reply:
I guess not. I've checked for uint types in syzkaller. But I've not been able to find any unsigned type.